homeassistant/components/flux/switch.py

```diff
@@ -320,6 +326,11 @@ def is_on(self):
                 time_state,
                 now,
             )
+            self._state_attributes = {}
+            self._state_attributes["Colour Temperature (K)"] = f"{int(temp)}"
+            self._state_attributes["Brightness (%)"] = f"{int(brightness / 255 * 100)}"
+            self._state_attributes["x, y"] = f"{round(x_val,4)}, {round(y_val,4)}"
```

Comment:
This is what @springstan meant in his initial comment.

Please adjust the attribute dict assignment/creation like this:

```suggestion
            self._state_attributes = {
                "Colour Temperature (K)" = f"{int(temp)}"
                "Brightness (%)" = f"{int(brightness / 255 * 100)}"
                "x, y" = f"{round(x_val,4)}, {round(y_val,4)}"
            }
```

Furthermore, the keys need to be lowercase, without spaces (underscore is allowed).
e.g., `color_temperature`, `brightness` & `x_y`/`xy`.

Please make sure to adjust this for all cases (including the 2 below).